adafruit_mpu6050.py

```diff
@@ -74,13 +75,36 @@
 STANDARD_GRAVITY = 9.80665
 
 
+class Clksel:  # pylint: disable=too-few-public-methods
+    """Allowed values for py:attr:`clock_source`.
+
+    * py:attr:'Clksel.CLKSEL_INTERNAL_8MHz
```

Comment:
It's an issue with `sphinx` building the documentation - many of the references are missing the leading colon (`:`)

```suggestion
    * :py:attr:'Clksel.CLKSEL_INTERNAL_8MHz
```

You'll need to do this for all the cases like this, I just tagged one of them.

Reply:
Ugh, okay.  Sorry I just missed this comment.  I genuinely appreciate the help.  

For my clarification - Sphinx is a tool/software that builds documentation based off these comments???  Just trying to understand the importance.  I could see automatic documentation generation being quite helpful.

I'll make those changes and re-commit... again haha

Reply:
Should i edit the original module Classes as well with the same changes?

Reply:
Yeah, `sphinx` is our tool that automatically builds the documentation from comments in the source code.  It's what ends up generating our documentation on ReadTheDocs.  `pre-commit` doesn't actually test building the documentation so that's why it can pass `pre-commit` but fail in the CI.  If you want to ever test building the documentation, you can do this from the repository folder once you've installed sphinx (`pip install sphinx`):

```shell
cd doc  # Make sure you're in the docs folder
sphinx-build -E -W -b html . _build/html
```

Reply:
For example: 

```class GyroRange:  # pylint: disable=too-few-public-methods
    """Allowed values for :py:attr:`gyro_range`.

    * :py:attr:`GyroRange.RANGE_250_DPS`
    * :py:attr:`GyroRange.RANGE_500_DPS`
    * :py:attr:`GyroRange.RANGE_1000_DPS`
    * :py:attr:`GyroRange.RANGE_2000_DPS`

    """
```